source/features/repo-header-info.tsx

```diff
@@ -53,7 +55,11 @@ async function add(repoLink: HTMLAnchorElement): Promise<void> {
 				title={`Repository starred by ${stargazerCount.toLocaleString('us')} people`}
 				className="d-flex flex-items-center flex-justify-center mr-1 gap-1 color-fg-muted"
 			>
-				<StarIcon className="ml-1" width={12} height={12} />
+				{
+					viewerHasStarred
+						? <StarFillIcon className="ml-1" color="var(--button-star-iconColor)" width={12} height={12} />
```

Comment:
`color`? Shouldn’t it be `fill`?

Reply:
<img width="1634" alt="image" src="https://github.com/user-attachments/assets/11cf0881-dfd9-4674-b8a2-e8802919e443">

Reply:
Worth adding a comment. 

Also… something is up with our JSX, there are extra props. This is likely due to sucrase and it might need further config.